.github/workflows/quickstart_ci.yml

```diff
@@ -121,7 +121,7 @@ jobs:
           echo "Building bootable jar..."
           mvn -fae clean package -Pbootable-jar
           echo "Starting bootable jar..."
-          mvn -f ${{ inputs.DEPLOYMENT_DIR }}/pom.xml wildfly-jar:start -Djar-file-name=${{ inputs.DEPLOYMENT_DIR }}/target/${{ inputs.QUICKSTART_PATH }}-bootable.jar -Dstartup-timeout=120
+          mvn -f ${{ inputs.DEPLOYMENT_DIR }}/pom.xml wildfly-jar:start -Djar-file-name=${{ inputs.DEPLOYMENT_DIR }}/target/${{ inputs.QUICKSTART_PATH }}-bootable.jar -Dstartup-timeout=120 -Dserver.host=${{ inputs.SERVER_PROVISIONING_SERVER_HOST }}
```

Comment:
Because LRA Participant 1 calls LRA Participant 2 through REST in order to propagate LRA context - https://github.com/wildfly/quickstart/pull/778/files#diff-f1391a74278417712aeae9c99e15cef9c524855c66d09c5756d82f420bc6095cR74.